.github/workflows/docker-image.yml

```diff
@@ -13,32 +13,28 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DOCKER_REPO: jeessy/ddns-go
-      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
       DOCKER_PLATFORMS: linux/amd64,linux/arm,linux/arm64
-      DOCKER_REGISTRY: ""
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
       - name: Set up Docker buildx
-        id: buildx
-        uses: crazy-max/ghaction-docker-buildx@v3
+        uses: docker/setup-buildx-action@v1
```

Comment:
To replace the `crazy-max/ghaction-docker-buildx@v3`, setup QEMU first and then setup buildx